admin-ui/app/utils/LicenseScreens/GenerateLicenseCard.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import Card from '@mui/material/Card'
+import CardActions from '@mui/material/CardActions'
+import CardContent from '@mui/material/CardContent'
+import Typography from '@mui/material/Typography'
+import { useDispatch, useSelector } from 'react-redux'
+import { generateTrialLicense } from '../../redux/actions'
+import { useMediaQuery } from '@mui/material'
+
+const GenerateLicenseCard = () => {
+  const dispatch = useDispatch()
+  const isMobileDevice = useMediaQuery('(max-width:680px)')
```

Comment:
we should not float CSS in components. Please follow the way the css have been managed in admin-ui.